README.md

```diff
@@ -19,7 +19,7 @@ The PWA depends on data files generated by Scripture App Builder. There is examp
 
 #### Example Data
 
--   Run `npm run extract:example <project_name>` to generate data from projects in the `test_data/projects`. For example, `web_gospels` is an SAB project name and `hanga` is a DAB project name.
+-   Run `npm run extract:example <project_name>` to generate data from projects in the `test_data/projects`. For example, `web_gospels` is an SAB project name and `hanga` is a DAB project name. Note, this requires java to either be on the path, or to have a JAVA_HOME env variable on windows
```

Comment:
We try not to do things that only work on one platform, so if we implement this, we would remove the comment about windows as well.